subsys/infinispan/src/main/conf/conf.d/cache-example.xml

```diff
@@ -0,0 +1,6 @@
+<infinispan>
```

Comment:
When/where is the cache-example.xml used?

Reply:
That's good question, it should be file per cache. e.g: Not Found Cache will be cache-nfc.xml. Then we can declare them in hotrod-client.properties.
`#infinispan.client.hotrod.cache.nfc.configuration_uri=file:/var/lib/ispn/cache-nfc.xml`

Since the config is not supported until 11.x, we need to programmatically to load the specific file for ispn 9.x now.

Reply:
is this xml to create remote cache? or just tells indy how to use a remote cache?

Reply:
When first access the specific cache, it will create the remote cache according to this xml config if there is no existing one. So it's used to create remote cache.

Reply:
I see. Thanks.